inference-engine/src/mkldnn_plugin/utils/shape_inference/static_dimension.cpp

```diff
@@ -0,0 +1,93 @@
+// Copyright (C) 2018-2021 Intel Corporation
```

Comment:
why static dimension and static shape are not part of `shape_inference` library?

Reply:
Because we don't want them in ngraph -- binary size for the builds without CPU plug-in would benefit from that. And we would only use StaticShape in CPU

Reply:
when we build static library we can put any code here - if it's not used in the final dll library or executable, it's removed by linker.